### ISSUE TITLE
Second inventory slot is stuck when in Refined Storage window

### ISSUE THREAD

**Issue by EMifnick** [NillerMedDild/Enigmatica6#3074]
### Modpack Version

0.5.11

### Describe your issue

When I open the refined storage crafting terminal, the second inventory slot on my hotbar is stuck, by which I mean:

- Items cannot be taken out of it
- Items cannot be shift-clicked out of it
- The item in it cannot be swapped with another or moved using other inventory hotkeys

But as soon as I am out of the RS window and in any other inventory window, it behaves completely normally and is usable again.

This bug was not present in 0.5.7, but updating to 0.5.11 created this bug. 
(also, it has been a lot laggier, but I do not know if that is due to the version, the Mek fission reactor I built around the same time, or the RedstonePen wiring I used on the reactor E-SCRAM system and in other places, as that is all around the same time. Still need to test that, and anyhow it's a different issue.)

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by Honhk** [comment]
You are using your wireless crafting grid in a Curios slot, right? Because it will disable a slot depending on where it goes (If in a slot there is only one of, the first, if in the second slot say for example a ring it will take the second slot, and so on.

**Comment by Darkere** [comment]
Yes and when used as bauble it will cause the second slot to get blocked... because bug :)
